specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/preview/2018-09-01/examples/BackupLocations/Prune.json

```diff
@@ -0,0 +1,28 @@
+{
+    "parameters": {
+      "subscriptionId": "8742f75d-84ae-4f1b-b22e-0e63dde0d6f4",
+      "resourceGroupName": "System.redmond",
+      "location": "redmond",
+      "api-version": "2018-09-01",
+      "OperationType": "DryRun"
+    },
+    "responses": {
+      "200": {
+        "body": {
+            "id":"/subscriptions/abadebb1-df95-427e-b188-6730a206bc86/resourceGroups/system.redmond/providers/Microsoft.Backup.Admin/backupLocations/redmond",
```

Comment:
remove fields except properties - you have swagger model validation errors due to this

Reply:
Fixed